framework/CodeInterfaces/RELAP5/Relap5Interface.py

```diff
@@ -258,7 +258,7 @@ def _evaluateOperators(self,**Kwargs):
         expression = expression.replace(var,str(Kwargs['SampledVars'][var]))
       expr = copy.copy(expression)
       for card in operator['cards']:
-        expr = expr.replace("%card%",operator['cardsValues'][card])
+        expr = copy.copy(expression).replace("%card%",operator['cardsValues'][card])
```

Comment:
What is this edit doing?
In line 259 you already have  expr = copy.copy(expression)

Reply:
Unfortunately that replace is supposed to be done on the original expression. If you have a number of cards > 1 the first replace (operation) works but for the other cards it does not since ```%card%``` has been already replaced. If we use the original expression, the operation works since the expression string is not "mutated"